aries-site/src/pages/templates/hub-and-spoke-navigation.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import Link from 'next/link';
+import { Anchor } from 'grommet';
+
+import { BulletedList, CardGrid, Meta, SubsectionText } from '../../components';
+import { ContentSection, Example, Layout, Subsection } from '../../layouts';
+import { getPageDetails, getRelatedContent, nameToPath } from '../../utils';
+
+const title = 'Hub and Spoke Navigation';
+const topic = 'Templates';
+const page = getPageDetails(title);
+const relatedContent = getRelatedContent(title);
+
+const HubSpokeNavigation = () => (
+  <Layout title={title}>
+    <Meta
+      title={title}
+      description={page.seoDescription}
+      canonicalUrl="https://design-system.hpe.design/templates/hub-and-spoke-navigation"
+    />
+    <ContentSection>
+      <Subsection name={title} level={1} topic={topic}>
+        <SubsectionText>{page.description}</SubsectionText>
+        <Example code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/dashboards/DashboardExample.js">
+          {}
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          Hub &amp; Spoke Navigation works well for applications primarily
+          concerned with task execution, as opposed to presenting large volumes
+          of content.
+        </SubsectionText>
+        <SubsectionText>
+          The "hub" cleanly presents multiple discreet application sections or
+          "spokes." Each "spoke" keeps the user focussed on the task at hand and
+          used independently from other spokes.
+        </SubsectionText>
+        <SubsectionText>
+          A hub should consist of no more than five to seven spokes &mdash; if
+          more paths are neededm subsequent hubs are always an option. Each step
```

Comment:
```suggestion
          more paths are needed subsequent hubs are always an option. Each step
```

Reply:
Fixing. That 'm' is supposed to be a comma.